src/extrinsic/utility.rs

```diff
@@ -21,7 +21,8 @@
 use crate::{rpc::Request, Api};
 use ac_compose_macros::compose_extrinsic;
 use ac_primitives::{
-	CallIndex, ExtrinsicParams, FrameSystemConfig, SignExtrinsic, UncheckedExtrinsicV4,
+	extrinsic_params::ExtrinsicParams, extrinsics::CallIndex, FrameSystemConfig, SignExtrinsic,
```

Comment:
To do this, we need to add an `extrinsics' module to the primitives:
- extrinsics/
   - mod.rs (CallIndex, UncheckedExtrinsicV4)
   - extrinsic_params.rs
   - signer.rs

And we will get: `use ac_primitives::extrinsics::{AssetTipExtrinsicParams, ExtrinsicParams, CallIndex, SignExtrinsic,UncheckedExtrinsicV4};`

This would not be consistent with the `RpcParams` use `ac_primitives::RpcParams`. Similarly to https://github.com/scs/substrate-api-client/pull/473#discussion_r1141937472